content/posts/akri-a-year-later.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Akri a Year Later"
+description: "Akri turns One and enters the CNCF Sandbox"
+date: 2021-10-20 00:00:00 +0000 UTC
```

Comment:
Before merge, you'll want to ensure this date matches the date you are publishing the blog post.

Reply:
@kate-goldenring to confirm, but I think we want to push it out on the 20th, if that's okay

Reply:
Oh, time zones..
I think the blog theme we are using doesn't render the actual time of the post, so the correct date should be enough.

Pinging @flynnduism just in case.